docs/_posts/0000-03-02-oo-encapsulation.md

```diff
@@ -177,19 +164,19 @@ Note: Conduct a straw poll and see how many classes they came up with.
 * You'll need to change the playing card class to use it
 * The test should still pass
 
-```text
+<mermaid>
+classDiagram
+    PlayingCard *-- Suit
```

Comment:
I suggest we don't include the association in the diagram as we've not mentioned that notation and hence is probably just a source of confusion:
```suggestion
    PlayingCard
```

Reply:
Yeah, we haven’t mentioned that notation but I don’t think it’s necessary to avoid mentioning it. I’ve got a note to talk about it, and I’ve added a label to the relationship in fce659a3267acb44cd0909cdaaaab878fd3abbca.